packages/utils/src/lib/file-system.ts

```diff
@@ -61,11 +62,10 @@ export function logMultipleFileResults(
   const succeededCallback = (result: PromiseFulfilledResult<FileResult>) => {
     const [fileName, size] = result.value;
     const formattedSize = size ? ` (${chalk.gray(formatBytes(size))})` : '';
-    console.info(`- ${chalk.bold(fileName)}${formattedSize}`);
-  };
-  const failedCallback = (result: PromiseRejectedResult) => {
-    console.warn(`- ${chalk.bold(result.reason)}`);
+    return `- ${chalk.bold(fileName)}${formattedSize}`;
   };
+  const failedCallback = (result: PromiseRejectedResult) =>
+    `- ${chalk.bold(result.reason)}`;
```

Comment:
Calling these _callbacks_ is now very misleading and looks like a mistake. Giving them a name like `getFailedMessage` would better indicate their purpose.

Reply:
mentioned it above that i renamed to xzyTransform